src/jit/codegencommon.cpp

```diff
@@ -4336,17 +4336,6 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
             }
         }
```

Comment:
Could you guide how to correct this issue? What I observed is that when HFA consists of 2 double elements (which occupies 4 registers), the base offset of the second argument becomes 16. In my understanding, it should be 8.